ci/provisioning/post_provision_config.sh

```diff
@@ -16,6 +16,11 @@ DSL_REPO_var="DAOS_STACK_${DISTRO}_LOCAL_REPO"
 DSG_REPO_var="DAOS_STACK_${DISTRO}_GROUP_REPO"
 
 clush -B -l root -w "$NODESTRING" -c ci_key* --dest=/tmp/
+git log --format=%s -n 1 HEAD | ssh root@${NODELIST%%,*} "cat >/tmp/commit_title"
+git log --pretty=format:%h --abbrev-commit |
+  ssh root@${NODELIST%%,*} "cat >/tmp/commit_list"
+ssh root@${NODELIST%%,*} "mkdir /scratch && mount wolf-2:/export/scratch /scratch"
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]